### DISCUSSION TITLE
No block reward

### DISCUSSION THREAD

**Discussion by Dieria14** [Chia-Network/chia-blockchain#5322] [Support]
Hello. There was such a problem 29.04.21 knocked out the block (at the height of 201641, address xch1j8vkxknuakd6lf2guefr8qpc3g9h0d3sujycq2u55zgkyw9q9egskjg3t2) But in the nodes tab, the wallet is on balance 0, there are receipts in the history. In the farm tab, there are also received coins. What could be the problem and where to turn?

Здравствуйте. Возникла такая проблема 29.04.21 выбил блок (на высоте 201641, адрес xch1j8vkxknuakd6lf2guefr8qpc3g9h0d3sujycq2u55zgkyw9q9egskjg3t2) Но во вкладке ноды кошелек на балансе 0, в истории есть поступления. Во вкладке ферма так же есть полученные монеты. В чем может быть проблема и куда обратиться?

![Cash](https://user-images.githubusercontent.com/84122851/118440934-dfa64c00-b701-11eb-9b79-9ed54d8312d6.JPG)
![Farm](https://user-images.githubusercontent.com/84122851/118440938-e1700f80-b701-11eb-9906-6e4e8208f55b.JPG)

**Comment by OversizedWizard** [comment]
Well that's not fair.  I have lots more plots than you, my remaining time is half Yours and I haven't found any rewards yet!!!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
